From 52d5c96ac034bdd9505e96ec2f9307d6b1a94c0c Mon Sep 17 00:00:00 2001
From: Michael Jeanson <mjeanson@efficios.com>
Date: Thu, 26 Jan 2017 14:54:19 -0500
Subject: [PATCH] Fix: Remove unused headers

This is a portability fix, these headers are unused and not available on
some platforms.

Signed-off-by: Michael Jeanson <mjeanson@efficios.com>
---
 src/bin/lttng-relayd/health-relayd.c | 1 -
 src/common/futex.c                   | 1 -
 2 files changed, 2 deletions(-)

diff --git a/src/bin/lttng-relayd/health-relayd.c b/src/bin/lttng-relayd/health-relayd.c
index cff6c6e..b14b827 100644
--- a/src/bin/lttng-relayd/health-relayd.c
+++ b/src/bin/lttng-relayd/health-relayd.c
@@ -37,7 +37,6 @@
 #include <sys/mman.h>
 #include <assert.h>
 #include <urcu/compiler.h>
-#include <ulimit.h>
 #include <inttypes.h>
 
 #include <common/defaults.h>
diff --git a/src/common/futex.c b/src/common/futex.c
index 2ae03b2..384e957 100644
--- a/src/common/futex.c
+++ b/src/common/futex.c
@@ -18,7 +18,6 @@
 
 #define _LGPL_SOURCE
 #include <limits.h>
-#include <sys/syscall.h>
 #include <unistd.h>
 #include <urcu.h>
 #include <urcu/futex.h>
